core/src/main/java/org/sonarsource/sonarlint/core/WsHelperImpl.java

```diff
@@ -19,6 +19,7 @@
  */
 package org.sonarsource.sonarlint.core;
 
+// TODO
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarlint.core%3Asonarlint-core-parent&branch=junk2&issues=AWEyAmflzwlWAzALxAcz&open=AWEyAmflzwlWAzALxAcz)

[ISSUE_KEY]: AWEyAmflzwlWAzALxAcz